File: 백준/Bronze/2309. 일곱 난쟁이/일곱 난쟁이.py
dwarf = []
sum=0
for i in range(9):
    dwarf.append(int(input()))
    sum += dwarf[i]
dwarf.sort()

for i in range(9):
    for j in range(1,9):
        if i==j: continue
        if sum-dwarf[i]-dwarf[j] == 100:
            for x in dwarf:
                if x==dwarf[i] or x==dwarf[j]: continue
                print(x)
            quit()

File: 백준/Bronze/2309. 일곱 난쟁이/일곱 난쟁이.cc
#include <iostream>
#include <algorithm>
using namespace std;

int main() {
    int dwarf[9];
    int sum = 0;
    
    for(int i=0;i<9;i++ )
    {
        int x,first,second;
        cin >> x;
        
        dwarf[i] = x;
    }
    for(int i = 0;i<9;i++)
    {
        sum+=dwarf[i];
    }
    sort(dwarf,dwarf+9);
    for(int i = 0; i < 9; i++)
    {
        for(int j=1;j<9;j++)
        {
            if(i==j) continue;
            if(sum-dwarf[i]-dwarf[j] == 100)
            {
                for(int x : dwarf)
                {
                    if(x==dwarf[i]||x==dwarf[j]) continue;
                   cout << x << '\n';
                }
                return 0;
            }
        }
    }

}